api/tests/src/api/agencies_v1/test_agencies_routes.py

```diff
@@ -10,7 +10,6 @@
 class TestAgenciesRoutes(BaseTestClass):
     @pytest.fixture(autouse=True)
     def cleanup_agencies(self, db_session):
-        yield
```

Comment:
Because the seed script test runs before this, it had agencies setup which broke these tests - made it so removing the yield makes this cleanup before the tests run, and solve that issue.